### ISSUE TITLE
Elements must only use allowed ARIA attributes for SVG g element with aria-label

### ISSUE THREAD

**Issue by AleksandrSl** [dequelabs/axe-core#3389]
Product: axe Extension

Sorry for the divergence from the template. It's actually more a question than a bug report.

I have an SVG with a group that renders text as paths and [animation library](https://github.com/airbnb/lottie-web) producing this SVG adds `aria-label` with this text to the group.
```
<svg >
  <g aria-label="some text" >...</g>
</svg>
```
Axe (and lighthouse) gives an error "Elements must only use allowed ARIA attributes". I've read a spec and your latest issue about prohibiting `aria-label` for some roles. The question is does `g` element has a `generic` or a `group` role. I've searched spec for information about `g` element, found [this one](https://www.w3.org/TR/2018/WD-svg-aam-1.0-20180510/) (it's old but the latest one). It says that `g` maps to a group role if I read it right and `group` role is allowed to have `aria-label`
![Screenshot from 2022-02-08 18-04-38](https://user-images.githubusercontent.com/15820496/153014585-6600d5c6-92e2-469f-8b58-cc5dc1830dde.png)

I think that in this case `title` should be used instead of `aria-label` and will ask the maintainers about that.
Though I'm still interested in the reason (partially as the argument to animation library not to use `aria-label`), whether it's something I don't see in specs or just some empiric rule that no screen readers will handle this the right way.

<pre><code>
axe-core version: v4.3.5
extension version: v4.22.1
</code></pre>

**Comment by alanfluff** [comment]
Hmm. I have an `svg` littered in `g` elements and they all/most have an `aria-label` attribute. But NVDA (Chrome, Win 10) simply does not see/read them.

Although [`g` is allowed to use `aria-label`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/g#aria_attributes), perhaps this is a case of the spec' allowing something, but it's uncommon and so SRs/NVDA only look for this attribute on the `svg` tag itself?

**Comment by alanfluff** [comment]
I assume this is relevant and may also be helpful https://stackoverflow.com/a/72658812
